Service-Broker-Setup.adoc

```diff
@@ -1,39 +1,36 @@
-# NOTE: This documentation assumes some future changes are in place
+== Service Broker Setup
+
+NOTE: This documentation assumes some future changes are in place
```

Comment:
If that should become a complete note with two phrases, a "full stop" is missing (... in place.).
Otherwise the second sentence should be separated from the note.